pkg/injector/pod_patch.go

```diff
@@ -723,8 +723,8 @@ func getSidecarContainer(annotations map[string]string, id, daprSidecarImage, im
 		SecurityContext: &corev1.SecurityContext{
 			AllowPrivilegeEscalation: &allowPrivilegeEscalation,
 		},
-		Ports:   ports,
-		Command: cmd,
```

Comment:
Why cmd moved to args?

Reply:
It's discussed in the comments above.

Essentially in K8s, Cmd is the ENTRYPOINT in Dockerfiles, and we need not to override that.